api/src/pcapi/core/bookings/models.py

```diff
@@ -331,6 +331,30 @@ def isExternal(cls) -> Label:  # pylint: disable=no-self-argument
             ).label("isExternal")
         ).label("number_of_externalBookings")
 
+    @hybrid_property
+    def validated_incident_id(self) -> int | None:
+        for incident in [booking_incident.incident for booking_incident in self.incidents]:
+            if incident.status == finance_models.IncidentStatus.VALIDATED:
```

Comment:
Pourquoi deux boucles ?
```suggestion
        for booking_incident in self.incidents:
            if booking_incident.incident.status == finance_models.IncidentStatus.VALIDATED:
```